src/Components/CardList/ListType/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { IList } from "./types"
+import Card from "../../SingleCard"
+
+const List = ({ type, todoList, setTodoList, completedTodos, setCompletedTodos, nonCompletedTodos, setNonCompletedTodos }: IList) => {
+    const getList = () => {
+        switch (type) {
+            case "allTasks":
+            default:
+                return (
+                    <div>
```

Comment:
[fix] Whenever you're rendering a list you should use either a `ul`, for lists in which the items have no order like this one, or a `ol`, for any list that has some order or hierarchy, like a list of steps in a recipe or a breadcrumb component. Also, you'll need to wrap the `Card` component with a `li` tag.

Reply:
Just did! Thank you Gustavo.